api/api.go

```diff
@@ -30,15 +30,18 @@ type Api struct {
 }
 
 type NaisDeploymentRequest struct {
-	Application string `json:"application"`
-	Version     string `json:"version"`
-	Environment string `json:"environment"`
-	Zone        string `json:"zone"`
-	ManifestUrl string `json:"manifesturl,omitempty"`
-	Username    string `json:"username"`
-	Password    string `json:"password"`
-	OnBehalfOf  string `json:"onbehalfof,omitempty"`
-	Namespace   string `json:"namespace"`
+	Application      string `json:"application"`
+	Version          string `json:"version"`
+	Zone             string `json:"zone"`
+	ManifestUrl      string `json:"manifesturl,omitempty"`
+	Environment      string `json:"environment"` // Deprecated: Use FasitEnvironment instead
+	Username         string `json:"username"`    // Deprecated: Use FasitUsername instead
+	Password         string `json:"password"`    // Deprecated: Use FasitPassword instead
```

Comment:
I think it's also good we are giving the proper prefix for these variables. So it's easy to understand which username and password we are supposed to use.